### ISSUE TITLE
Ud

### ISSUE THREAD

**Issue by SanderH030** [Dielee/volvo2mqtt#156]
A couple of weeks ago, Volvo implemented an update in the app. After this update, the temperature is also visible in the app. I assume that this temperature sensor originates from the car itself. The sensor is also capable of deciding whether to heat or cool based on the temperature. Will this sensor be made available over MQTT as well? I've created some automations for my car, which are currently all based on the outside temperature at my home. Having access to the car's own temperature would be a significant enhancement.

Thank you in advance :)

**Comment by PoLoMoTo** [comment]
I don't think this is actually a sensor, at least not on my 2023 V60 T8. It is currently parked in my garage and says it is 24F. I also happen to have a temperature sensor in my garage that is currently reading 48F, albeit it is mounted a little bit higher than the top of the car so it could be colder lower buuuuuut 24 degrees? Idk. Even still, it isn't even that cold outside, I happen to also have a temperature sensor outside that is currently reporting 29F and the last time the car was outside a few hours ago it most certainly would have been even warmer. My only thought is that they are grabbing that temperature from a weather service based on the cars location, at least for my car right now.

**Comment by Dielee** [comment]
They removed the API endpoint for the outside temperature in the last change. So there is no way to get this information, sorry!
